input/fsh/valuesets/au-erequesting-ranzcr-radiology-procedures.fsh

```diff
@@ -1,159 +1,160 @@
 ValueSet: RANZCRReferralProcedures
 Id: ranzcr-referral-procedures
 Title: "RANZCR Referral Procedures"
-Description: "Standard codes for use in requesting radiology tests in Australia, derived from the RANZCR Referral Set (RRS)."
+Description: "Standard codes for use in requesting radiology tests in Australia, derived from the RANZCR Radiology Referral Set (RRS)."
+* ^meta.profile[+] = "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
 * ^version = "1.0.0"
 * ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
 * ^status = #draft
 * ^experimental = false
 * ^date = "2024-07-17"
 * ^publisher = "HL7 Australia"
-* $sct#45036003 "Ultrasonography of abdomen"
-* $sct#444900008 "Ultrasonography of abdomen with contrast"
+* $sct#45036003 "Ultrasound of abdomen"
+* $sct#444900008 "Ultrasound of abdomen with contrast"
 * $sct#418394000 "Ultrasound scan of abdomen and pelvis"
-* $sct#241462009 "Ultrasound scan of abdominal aorta"
-* $sct#241512001 "Ultrasonography of Achilles tendon"
-* $sct#871000087105 "Ultrasonography of left Achilles tendon"
-* $sct#881000087107 "Ultrasonography of right Achilles tendon"
-* $sct#1921000087100 "Ultrasonography of bilateral Achilles tendons"
-* $sct#431844005 "Ultrasonography of acromioclavicular joint"
+* $sct#241462009 "Ultrasound of abdominal aorta"
+* $sct#241512001 "Ultrasound of Achilles tendon"
+* $sct#871000087105 "Ultrasound of left Achilles tendon"
+* $sct#881000087107 "Ultrasound of right Achilles tendon"
+* $sct#1921000087100 "Ultrasound of bilateral Achilles tendons"
```

Comment:
I notice in line 28 we've used the synonym of "left and right ankles" but bilateral for the achilles tendon
Do we want to change all Bilat to left and right where synonym allowed?

Ultrasound of left and right Achilles tendon - Synonym per international browser

Reply:
Hi @h4nsec , Those displays are the AU Preferred terms so let's discuss at our next catchup with Matt C. There may be editorial rules around this for AU Content we need to modify.

Reply:
Sounds good, thanks Michael